src/garage/_dtypes.py

```diff
@@ -919,3 +921,28 @@ def from_time_step_list(cls, env_spec, ts_samples):
         ]
 
         return TimeStepBatch.concatenate(*ts_batches)
+
+    @classmethod
+    def from_trajectory_batch(cls, batch):
+        """Construct a TimeStepBatch from a TrajectoryBatch.
+
+        Args:
+            batch (TrajectoryBatch): TrajectoryBatch to convert.
```

Comment:
please document shapes

Reply:
what shapes? the dtype is well-defined. that's why we defined it! to stop typing shapes all the time.